HCaptcha/Classes/HCaptchaVerifyParams.swift

```diff
@@ -69,14 +100,17 @@ public class HCaptchaVerifyParams: NSObject {
     public func toDictionary() -> [String: Any] {
         var dict: [String: Any] = [:]
         if let phonePrefix = phonePrefix {
-            dict["phonePrefix"] = phonePrefix
+            dict["mfa_phoneprefix"] = phonePrefix
         }
         if let phoneNumber = phoneNumber {
-            dict["phoneNumber"] = phoneNumber
+            dict["mfa_phonenumber"] = phoneNumber
         }
         if let rqdata = rqdata {
             dict["rqdata"] = rqdata
         }
+        if let userJourneys = userJourneys {
+            dict["user_journeys"] = userJourneys
```

Comment:
Mind if we use the camel case method like you have as the variable already? We do it for every other parameter, although MFA ones seemed to have deviated some how.